crates/hyperswitch_domain_models/src/payments.rs

```
@@ -799,7 +801,14 @@ where
                     },
                 payment_method_type: self.payment_attempt.payment_method_type,
                 payment_method_subtype: self.payment_attempt.payment_method_subtype,
-            });
+                connector: connector
+                    .parse()
+                    .map_err(|_| errors::api_error_response::ApiErrorResponse::InternalServerError)
```

Avoid discarding the error, log the error